lib/WP_Auth0_Api_Client.php

```diff
@@ -305,6 +305,7 @@ public static function create_client( $domain, $app_token, $name ) {
 
 			'callbacks'           => array(
 				$options->get_wp_auth0_url(),
+				$options->get_wp_auth0_url(),
```

Comment:
why does this array contains the same url twice?

Reply:
Odd ... that should not be a part of this PR. I'll remove it.